packages/validators-util/src/isPhoneNr/isPhoneNr.ts

```diff
@@ -0,0 +1,3 @@
+const PHONENR_REGEX = /^\d{8}$/;
+
+export const isPhoneNr = (value: string) => PHONENR_REGEX.test(value);
```

Comment:
Tror jeg ville kalt den `isPhoneNumber`, jeg syns det er lettere og lese. Koden kan vi minifisere så vi sparer bytene, og intellisense fikser resten, så ingen grunn til å spare på tegnene :dancers:

Reply:
Jag syns Nr precis som Org, är ganska ganska enkla och förhålla sig till i ett context sammanhang, exempel: OrgNr, den är super vanlig å se i finansiell kode, å alla som jobbar i domänet vet vad det betyder. Jag syns det blir svårare att gå igenom kode, datamodeller när allt må skrivas ut, eg: "NorwegianLimitedOrganisationNumber", hade jag föredragit som NOLimitedOrgNr.  Så länge alla känner till domänet no issues, men de som ikke kan, må förklaras, min synsning är, let it be, let it be :)

Reply:
Akkurat "number" er litt spesielt på engelsk; det skal kun forkortes når man snakker om et nummer i en rekke (f.eks. "no. 42"), ikke i setninger. Om man skal forholde seg til den regelen i variabelnavn er jo en annen sak. Dersom vi skal forkorte det foreslår jeg imidlertid å heller bruke `isPhoneNum` så vi ikke blander norsk og engelsk.

Reply:
Liker også `num` bedre enn `nr`